runtime/compiler/CMakeLists.txt

```diff
@@ -49,6 +49,12 @@ if(OMR_ARCH_X86)
 	omr_append_flags(CMAKE_ASM_NASM_FLAGS ${asm_inc_dirs})
 endif()
 
+# This target is created so that various subdirectories can add dependencies to it
```

Comment:
Seems this line should end with a period, and the next line start with a capital.